vulnfeeds/utility/logginghelpers.go

```diff
@@ -14,16 +14,19 @@ import (
 // and also returns a cleanup function to be deferred
 func CreateLoggerWrapper(name string) (LoggerWrapper, func()) {
 	projectId, ok := os.LookupEnv("GOOGLE_CLOUD_PROJECT")
+	var loggerOptions []logging.LoggerOption
 	if !ok {
-		log.Fatalf("GOOGLE_CLOUD_PROJECT not set")
+		log.Println("GOOGLE_CLOUD_PROJECT not set, routing logs to stdout")
+		loggerOptions = append(loggerOptions, logging.RedirectAsJSON(os.Stdout))
```

Comment:
I think this will be a line of JSON and not necessarily very locally consumer friendly.

Looking at https://pkg.go.dev/cloud.google.com/go/logging#hdr-The_Standard_Logger I'm wondering if it could return a standard https://pkg.go.dev/log#Logger instead?